### PR TITLE
Added get USD value snapshots endpoint

### PR DESCRIPTION
**Pull request by summersum-1**
## Summary
Added get USD value snapshots endpoint

## Additional Information
N/A

## Checks
- [ ] Changes tested locally (if applicable)
- [ ] Updated related documentation (if applicable)
- [ ] Updated/added related tests (if applicable)

### CODE REVIEW

File: src/rest-client.ts [hunk 1]
```diff
@@ -96,6 +96,10 @@ export class RestClient {
   acceptDust(quoteId: string): GenericAPIResponse {
     return this.requestWrapper.post(`dust/quotes/${quoteId}/accept`);
   }
+  
+  getUsdValueSnapshots(limit: number = 1000): GenericAPIResponse {
+    return this.requestWrapper.get(`wallet/usd_value_snapshots?${limit}`);
+  }
```
Comment (tiagosiebler) [inline src/rest-client.ts:102]: I don't see this in the docs, does it actually work? I don't mind adding it if it does but would like a jsdoc that it's undocumented, e.g:
```suggestion
  /**
   * Returns .......
   * Note: undocumented.
   */
  getUsdValueSnapshots(limit: number = 1000): GenericAPIResponse {
    return this.requestWrapper.get(`wallet/usd_value_snapshots?${limit}`);
  }
```

Also, the limit param, should the URL become `..../wallet/usd_value_snapshots/?1000` or should it be `..../wallet/usd_value_snapshots/?limit=1000`?

If it's the second one, you can do this instead - the connector will do the rest
```
    return this.requestWrapper.get(`wallet/usd_value_snapshots`, { limit });
```

Comment (summersum-1) [reply src/rest-client.ts:102]: I see it works and is used in the web UI. No, it's not documented. I'll add a jsdoc. 

Neat. I'll update how params are passed to the connector.

Comment (summersum-1) [reply src/rest-client.ts]: @tiagosiebler actioned

**Review by tiagosiebler — APPROVED** [review]
TIL about numeric separators for numbers in js - nice. Thanks again, happy to have undocumented endpoints (already added a few) as long as it's obvious it's undocumented.

**Comment by summersum-1** [comment]
Happy to contribute 🫂

**Pull request merged**